packages/relative-values/models/src/animals.ts

```diff
@@ -0,0 +1,617 @@
+import { Model } from "@/model/utils";
+import { Catalog, InterfaceWithModels, Item } from "@/types";
+import { sq } from "@quri/squiggle-lang";
+
+// Note: Much of this was generated by this file:
+// https://observablehq.com/d/98cbe0073226a5f9
+
+function getCatalog(): Catalog {
+  const items: Item[] = [
+    {
+      id: "cow_value_human_qalys_per_year_of_life",
+      name: "cow value per year of life",
```

Comment:
you don't have to say "value" everywhere, everything put in this app is a value.

"Average cow lifetime" or something. Can also be specific, like, "1 average(mean) cow life, unfarmed, 2022"

Reply:
You could also add a description here.